src/pkg/immutabletag/dao/immutable.go

```diff
@@ -76,7 +76,7 @@ func (i *immutableRuleDao) QueryImmutableRuleByProjectID(projectID int64) ([]mod
 // QueryEnabledImmutableRuleByProjectID get all enabled immutable rule by project
 func (i *immutableRuleDao) QueryEnabledImmutableRuleByProjectID(projectID int64) ([]model.ImmutableRule, error) {
 	o := dao.GetOrmer()
-	qs := o.QueryTable(&model.ImmutableRule{}).Filter("ProjectID", projectID).Filter("Enabled", true)
+	qs := o.QueryTable(&model.ImmutableRule{}).Filter("ProjectID", projectID).Filter("Disabled", true)
```

Comment:
the filter should changed to disabled=false

Reply:
ok, thanks.